### PR TITLE
Changes to Advisor deactivated recommendation email template

### PR DESCRIPTION
**Pull request by mhuth**

### CODE REVIEW

**Comment by gwenneg** [comment]
A few changes are required to use `{environment.url}` in that kind of test as it's not available by default while testing.

First, you need to add an injection to the test class:
```
@Inject
Environment environment;
```

Then, before the template is rendered, the `environment` instance must be passed to the template instance:

**Before**
```
        String result = Advisor.Templates.newRecommendationInstantEmailTitle()
                .data("action", action)
                .render();
```

**After**
```
        String result = Advisor.Templates.newRecommendationInstantEmailTitle()
                .data("action", action)
                .data("environment", environment)
                .render();
```

**Comment by gwenneg** [comment]
@mhuth How much time do you need to test this on stage? I could merge the PR now (the failed GH check is unrelated) but if I do, the changes may be deployed to prod shortly because we are deployed to prod today.

File: engine/src/main/resources/templates/Advisor/insightsEmailBody.html [hunk 1]
```diff
@@ -560,7 +560,7 @@
                                 </tr>
                                 <tr>
                                     <td class="rh-masthead__subhead">
-                                        <h1><b>{#insert content-title}{/}</b></h1>
+                                        <h1><b style="line-height: 1.5;">{#insert content-title}{/}</b></h1>
```
Comment (gwenneg) [inline engine/src/main/resources/templates/Advisor/insightsEmailBody.html:563]: I didn't know `style` worked on a `b` tag :smiley:

**Review by gwenneg — APPROVED** [review]
LGTM

File: engine/src/main/resources/templates/Advisor/insightsEmailBody.html (hunk 1 above)
Comment (mhuth) [reply engine/src/main/resources/templates/Advisor/insightsEmailBody.html:563]: It seemed to work :)  But I'll change it to the h1.

**Comment by mhuth** [comment]
@gwenneg yes feel free to go ahead and merge it so it makes it into prod.  These notifications don't happen very often anyway.

**Review by gwenneg — APPROVED** [review]

**Comment by mhuth** [comment]
@gwenneg I'm not sure about the failing tests though.  They seem to be unrelated to my changes.

**Comment by gwenneg** [comment]
That's unrelated indeed. We have an issue with QA tests today.

**Pull request merged**